### DISCUSSION TITLE
How to configure deployment in event source/sensor?

### DISCUSSION THREAD

**Discussion by anton-erofeev** [argoproj/argo-workflows#13409] [Q&A] [answered]
event source/sensor charts automatically create deployment. How can I configure this deployment? I need to change spec.templpate.spec for created deployment in sensor and event-source

**Comment by agilgur5** [comment, marked as answer]
This sounds like an Argo Events question, not a Workflows question.

It looks like you already asked in https://github.com/argoproj/argo-events/discussions/3221, that's the correct place to ask
